### PR TITLE
remaked

### PR DESCRIPTION
**Pull request by savchuksavchuk**
[DEMO LINK](https://savchuksavchuk.github.io/react_todo-app-loading-todos/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,223 +1,85 @@
...
-        </section>
+  useEffect(() => {
+    const filteredTodos = todos.filter(todo => {
+      switch (filter) {
+        case 'active':
+          return todo.completed === false;
+        case 'completed':
+          return todo.completed === true;
+        default:
+          return true;
```
Comment (vlad-demchuk) [inline src/App.tsx:57]: use `enum` for status

File: src/App.tsx [hunk 2]
```diff
@@ -1,223 +1,85 @@
...
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [visibleTodos, setVisibleTodos] = useState<Todo[]>([]);
+  const [isLoaded, setIsLoaded] = useState(false);
+  const [error, setError] = useState('');
```
Comment (vlad-demchuk) [inline src/App.tsx:20]: use more meaningful name for `error`.  eg `errorMessage`

File: src/App.tsx [hunk 3]
```diff
@@ -1,223 +1,85 @@
...
+    } catch {
+      setError('Unable to load todos');
+    }
 
-            <div data-cy="TodoLoader" className="modal overlay">
-              <div className="modal-background has-background-white-ter" />
-              <div className="loader" />
-            </div>
-          </div>
+    setIsLoaded(true);
```
Comment (vlad-demchuk) [inline src/App.tsx:38]: use `finally`

File: src/components/TodoItem.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+      {/* <form>
+            <input
+              data-cy="TodoTitleField"
+              type="text"
+              className="todo__title-field"
+              placeholder="Empty todo will be deleted"
+              defaultValue="JS"
+            />
+          </form> */}
```
Comment (vlad-demchuk) [inline src/components/TodoItem.tsx:44]: remove comments

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]

**Review by loralevitska — APPROVED** [review]
Good job!

try to fix such errors) 
image

**Pull request closed without merging**